homeassistant/components/androidtv/media_player.py

```diff
@@ -447,26 +447,20 @@ def __init__(
 
         # Property attributes
         self._adb_response = None
-        self._available = True
-        self._current_app = None
+        self._attr_available = True
+        self._attr_current_app = None
         self._sources = None
         self._state = None
         self._hdmi_input = None
 
-    @property
-    def app_id(self):
-        """Return the current app."""
-        return self._current_app
-
-    @property
-    def app_name(self):
-        """Return the friendly name of the current app."""
-        return self._app_id_to_name.get(self._current_app, self._current_app)
-
-    @property
-    def available(self):
-        """Return whether or not the ADB connection is valid."""
-        return self._available
+    async def async_update(self):
```

Comment:
I'm not sure if this works. Moving remaining attributes fails tests.

Reply:
It can't work because this function is never called, it's overridden by `async_update` in the `AndroidTVDevice` and `FireTVDevice` sub classes below.